HousingRepairsOnlineApi/SoRConfig.json

```diff
@@ -5,8 +5,9 @@
       "doorMissing":"N373049"
     },
     "worktop": "N372005",
-    "dampMould":{
-      "dampMould":"N114001"
+    "dampOrMould": {
+      "dampMouldCausedByLeak":"",
```

Comment:
This `dampMouldCausedByLeak` and the others shouldn't be present as this journey leads to an exit page.
Please remove them.

Reply:
I forgot to commit where i removed this!